sdk/entra/functions-authentication-events/review/functions-authentication-events.api.md

```diff
@@ -0,0 +1,144 @@
+## API Report File for "@azure/functions-authentication-events"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public (undocumented)
+export interface ActionableCloudEventResponse<TEventAction extends AuthEventAction> extends ActionableResponse<TEventAction> {
+    // (undocumented)
+    oDataType: string;
```

Comment:
can we lowercase odata? `odataType`?

Reply:
if possible, apply this to all other oData in the library

Reply:
Hey @joheredi , unfortunately we cannot, this is how it comes from the main library.